calvinextras/calvinsys/web/mqtt/Subscribe.py

```diff
@@ -219,10 +237,11 @@ def on_log_debug(client, string):
 
         _log.info("TLS: {}".format(tls))
         self.payload_only = payload_only
-        self.topics = [(topic.encode("ascii"), qos) for topic in topics]
+        self.topics = {(topic.encode("ascii")) : qos for topic in list(set(topics))}
         self.data = []
+        clean_session = kwargs.get('clean_session', 'false').lower() == "true"
```

Comment:
Clean session that comes in should already be a boolean.

Reply:
Done